executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -218,12 +216,17 @@ impl<'a, T: FieldElement> MachineExtractor<'a, T> {
         // receiving end of a lookup, we remove any multiplicity columns here.
         // Note that we don't use the passed identities, because we want to
         // include removed range constraints.
-        let multiplicity_columns = self
-            .fixed
-            .analyzed
-            .identities
+        let multiplicity_columns = convert(&self.fixed.analyzed.identities)
```

Comment:
Is there a way to have only one call to `convert`?

Reply:
I added code to filter polynomial identities, which should be most.